fortran/src/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if (WIN32)
       endif ()
       set (H5_F03EXP ";")
       if (NOT H5_FORTRAN_C_LONG_DOUBLE_IS_UNIQUE)
-        set (H5_FORTRAN_C_LONG_DOUBLE_IS_UNIQUE ";")
+        set (H5_FORTRAN_C_LONG_DBLE_IS_UNIQUE ";")
```

Comment:
This should work, but can the name not be so similar? Something short and with EXP for export?